util/PythonCommon.h

```diff
@@ -0,0 +1,50 @@
+#ifndef __FreeOrion__Util__PythonCommon__
+#define __FreeOrion__Util__PythonCommon__
+
+#include <boost/python.hpp>
+
+#include "Export.h"
+
+class FO_COMMON_API PythonCommon {
+public:
+    PythonCommon();
+    virtual ~PythonCommon();
+
+    /**
+       IsPythonRunning returns true is the python interpreter is
+       initialized.  It is typically called after
+       HandleErrorAlreadySet() to determine if the error caused the
+       interpreter to shutdown.
+     */
+    bool         IsPythonRunning() const;
+
+    /**
+       Handles boost::python::error_already_set.
+       If the error is SystemExit the python interpreter is finalized
+       and no longer available.
+
+       Call PyErr_Print() if the exception is an error.
+
+       HandleErrorAlreadySet is idempotent, calling it multiple times
+       won't crash or hang the process.
+     */
+    void HandleErrorAlreadySet();
+
+    bool         Initialize();        // initializes and runs the Python interpreter, prepares the Python environment
```

Comment:
Can you move the comment to a function name?
`InitializeInterpretorAndPreparePythonEnvironmet` is a little bit verbose, but we can get rid of the comment.

If you put the documentation into the code objects you'll get self-documenting code.

Reply:
In general yes, but in this case, does it really clarify anything more usefully with the longer name...?

Reply:
We have at least 2 places in code declaration and usage. And only one place has this comment. 

So the long name will spread the documentation along with the code.